spinalcordtoolbox/mocoDL/inference.py

```diff
@@ -0,0 +1,182 @@
+# -*- coding: utf-8 -*-
+"""
+Inference script used to generate the motion-corrected 4D volumes from DenseRigid model.
+"""
```

Comment:
Just a quick comment: for new Python files, their headers should follow the standard from #4128.

Also, these `coding: utf-8` comments are not required since Python 3 ([link](https://peps.python.org/pep-3120/)).